Library/Homebrew/dev-cmd/audit.rb

```diff
@@ -398,6 +398,7 @@ def audit_license
         return if user.blank?
 
         github_license = GitHub.get_repo_license(user, repo)
+        return unless github_license
         return if github_license && (licenses + ["NOASSERTION"]).include?(github_license)
```

Comment:
```suggestion
        return if (licenses + ["NOASSERTION"]).include?(github_license)
```

No need to check this twice anymore